rest-api-spec/src/main/resources/rest-api-spec/api/exists.json

```diff
@@ -42,6 +46,27 @@
         "routing": {
           "type" : "string",
           "description" : "Specific routing value"
+        },
+        "_source": {
+          "type" : "list",
+          "description" : "True or false to return the _source field or not, or a list of fields to return"
+        },
+        "_source_exclude": {
+          "type" : "list",
+          "description" : "A list of fields to exclude from the returned _source field"
+        },
+        "_source_include": {
+          "type" : "list",
+          "description" : "A list of fields to extract and return from the _source field"
+        },
```

Comment:
sorry for being late to the party, but I think these params weren't previously accepted by the exists api. I am wondering what their meaning is in the context of an exists request, Version makes sense to me, but the exclude, include and source don't. Note that we have a separate endpoint (`/{index}/{type}/{id}/_source`) to check whether the source exists or not (which is not in our spec and that is a bug that I will fix). Thoughts?